core/src/main/java/io/grpc/transport/HttpUtil.java

```diff
@@ -69,6 +69,13 @@
   public static final String TE_TRAILERS = "trailers";
 
   /**
+   * The User-Agent header named. Defined here since it is not explicitly defined by the HTTP/2
```

Comment:
I'm not sure I understand this comment. Won't users need to access this? Don't we want it in a more public location?